### ISSUE TITLE
Error in default route construction crashes chrome.

### ISSUE THREAD

**Issue by Zoltu** [aurelia/router#137]
If your default route redirects and you have an error in the redirected page the router will redirect back to the default route which will redirect back to the page which will error again.  This loops forever and Chrome does not handle infinite anchor redirects well and the entire browser locks up (not just the tab).

On a somewhat related note, there should be a way to tell the router about an error page that it can redirect to rather than redirecting back to default when a view fails to construct.

**Comment by Zoltu** [comment]
```js
export class App {
	configureRouter(config, router) {
		config.map([{ route: '', redirect: 'crash' }]);
	}
}
```
```js
export class Crash {
	constructor() {
		throw new Error("Rawr!");
	}
}
```

**Comment by bryanrsmith** [comment]
Thanks, @Zoltu. This was fixed by #130, which will be included in the next release.
